modes/basic-dev-mode/src/toolbarButtons.js

```diff
@@ -34,7 +31,7 @@ function _createSetToolActiveCommands(toolName, toolGroupIds = ['default', 'mpr'
   }));
 }
 
-const toolbarButtons: Button[] = [
+const toolbarButtons = [
```

Comment:
Please rename the file to toolbarButtons.ts instead of removing the type values.